python/lsst/obs/decam/decamMapper.py

```diff
@@ -113,10 +116,10 @@ def _computeCoaddExposureId(self, dataId, singleFilter):
                                    filter coadd, in which case dataId
                                    must contain filter.
         """
-        tract = long(dataId['tract'])
+        tract = int(dataId['tract'])
         if tract < 0 or tract >= 2**DecamMapper._nbit_tract:
             raise RuntimeError('tract not in range [0,%d)' % (2**DecamMapper._nbit_tract))
-        patchX, patchY = map(int, dataId['patch'].split(','))
+        patchX, patchY = [int(x) for x in dataId['patch'].split(',')]
```

Comment:
Yes, map functionality can be done with list comprehensions, and it's usually easier to read. There was once a speed difference (circa python 2.4), but that's no longer the case.